README.md

```diff
@@ -0,0 +1,9 @@
+# Spy Terminal
```

Comment:
A README file is a helpful way to understand the purpose of a repository, how to get started running it locally, how the program is hosted and how it's deployed, maybe guidelines for contributing, etc. Just generally a helpful document for another developer to be able to contribute. I gave it a start, but there's some encryption key or file I think that I would need to run your program on my machine.